vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -6,7 +6,7 @@
 	<body>
 		<div th:fragment="navbar">
 			<nav class="nav-bar" style="height:48px">
-		      <a class="navbar-brand" href="#"><img th:src="@{/resources/maintenance/img/logo-11.png}" alt="Virtual Spaces logo" width="100px">
+		      <a class="navbar-brand" th:href="@{|/exhibit/space/${exhibition?.startSpace?.id}|}"><img th:src="@{/resources/maintenance/img/logo-11.png}" alt="Virtual Spaces logo" width="100px">

```

Comment:
This will create issues if there is no exhibition set up yet. Have you tried this: 
https://github.com/diging/virtual-spaces-2.0/blob/e6a8ea2493b535e366bae49d4e74a55c01624f3a/vspace/src/main/webapp/WEB-INF/views/layouts/navbar_staff.html#L64